hal/include/HAL/HAL.hpp

```diff
@@ -234,6 +239,7 @@ extern "C"
 	int HALGetControlWord(HALControlWord *data);
 	int HALGetAllianceStation(enum HALAllianceStationID *allianceStation);
 	int HALGetJoystickAxes(uint8_t joystickNum, HALJoystickAxes *axes);
+  int HALGetJoystickAxesFloat(uint8_t joystickNum, HALJoystickAxes *axes, HALJoystickAxesFloat *axesFloat);
```

Comment:
I was using spaces since the style guide PR is going to change this entire thing to spaces, and I forgot to change my spacing back.